web/frontend/src/components/slam.vue

```diff
@@ -22,7 +21,13 @@ const props = defineProps<{
   resources: commonApi.ResourceName.AsObject[]
   client: Client
   statusStream: ResponseStream<robotApi.StreamStatusResponse> | null
-}>();
+  operations: {
+    op: robotApi.Operation.AsObject
```

Comment:
What is this doing?

Reply:
this adds all the current operations a robot is doing